app/workers/podcasts/get_episodes_worker.rb

```diff
@@ -0,0 +1,22 @@
+module Podcasts
+  class GetEpisodesWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority, retry: 10
+
+    def perform(podcast_data = {})
```

Comment:
Using a Hash here since there are more than 2 arguments and it's not clear when calling the worker what the arguments are (i.e. passing in just a number `5`).